README.md

```diff
@@ -1,2 +1,34 @@
 # Hello-Microverse
+
 A Hello Microverse project with concepts like Github Flow, documenting work in professional way, and feature branching
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Description abou my hello microverse project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+
+- [👥 Authors](#authors)
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hell Microverse]** is a a simple task to master all of the tools and best practices I learned about linter, githubflow and documenting readme professionally.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
+
+- GitHub: [@fithamlak](https://github.com/fithamlak)
```

Comment:
Great work so far with the requested changes. however as requested by the previous reviewer, kindly use the microverse [readme template](). Its a great effort you have some of the elements the template has, but to make your project better and be ablt to showcase your work better, I suggest you copy the template as it is and edit it to fit this project

To copy, click on the copy button on the template as shown below by below arrow:
<img width="1665" alt="Screenshot 2022-11-30 at 16 40 02" src="https://user-images.githubusercontent.com/48546992/204811508-c8a2801d-63a1-477c-8102-57a3f4a607d0.png">